lib/rules/no-invalid-double-slash-comments/index.mjs

```diff
@@ -46,19 +49,19 @@ const rule = (primary) => {
 					const offset = value.length - selector.length;
 					const i = index + offset;
 
-					ruleStringified ??= ruleNode.toString();
-					const lines = ruleStringified.slice(i).split('\n');
-					let comment = lines[0] ?? '';
+					ruleStringified ??= ruleNode.source.input.css;
 
-					if (lines.length > 1) comment += '\n';
+					const lines = ruleStringified.slice(i).split('\n');
+					const comment = lines[0] ?? '';
```

Comment:
note

TS is overzealous.
i.e. `comment[0]` can be `undefined` only if `''` is passed to `split`
e.g. `''.split('')`